charts/sps-sample/values.yaml

```
@@ -126,3 +161,150 @@
+  auth:
+    username: user
+    password: admin
+    database: db
+  primary:
+    podSecurityContext:
+      fsGroup: null
+    containerSecurityContext:
+      runAsUser: null
+      runAsNonRoot: true
+      allowPrivilegeEscalation: false
+      seccompProfile: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)